provision/acc_provision/apic_provision.py

```diff
@@ -1022,6 +1032,14 @@ def kube_tn(self):
                                     }
                                 }
                             ]
+                        },
+                        "ndPfxPol": {
+                            "attributes": {
+                                "ctrl": "on-link,router-address",
+                                "lifetime": "2592000",
+                                "name": "kube-nd-%s" % system_id,
```

Comment:
Naming convention for these is e.g.:
l3out_rsprov_name = "%s-l3out-allow-all" % system_id
Except that this is needs to be done for things in the common tenant.  For stuff in the "system_id" tenant like this you can just use a sensible fixed name.  Probably something like simple and obvious like "kube-nd-ra-policy".  Also, only add for ipv6 subnets; don't change for ipv4 subnets.

Reply:
Done, I have uploaded the new changes.